src/main/java/it/gov/pagopa/reward/dto/mapper/TransactionProcessed2SyncTrxResponseDTOMapper.java

```diff
@@ -18,6 +18,9 @@ public SynchronousTransactionResponseDTO apply(BaseTransactionProcessed baseTran
         out.setChannel(trx.getChannel());
         out.setInitiativeId(initiativeId);
         out.setUserId(trx.getUserId());
+        out.setOperationType(baseTransactionProcessed.getOperationTypeTranscoded());
+        out.setAmount(baseTransactionProcessed.getAmountCents());
```

Comment:
let's return amount not amountCents

every currency in other fields returned are euro (effectiveAmount, reward.accruedReward)
```suggestion
        out.setAmount(baseTransactionProcessed.getAmount());
```